components/HistoryMessages.tsx

```diff
@@ -6,25 +6,36 @@ export default function HistoryMessages({
   messages,
   user,
 }: {
-    messages: Message[];
-    user: User;
+  messages: Message[];
+  user: User;
 }) {
+  const formatTimestamp = (timestamp) => {
+    const dateObject = new Date(timestamp);
+    const hours = dateObject.getHours().toString().padStart(2, "0");
+    const minutes = dateObject.getMinutes().toString().padStart(2, "0");
+    return `${hours}:${minutes}`;
+  };
+
   return (
-    <div>
-      <h1>All Messages</h1>
-      {messages?.map((message, index) => {
-        if (message.senderId === user.id)
-          return (
-            <div className="text-right text-green-300" key={index}>
-              {message.messageContent}
+    <div className="w-full h-full flex flex-col items-stretch">
+      <div className="bg-teal-500 text-white py-2 text-center">
+        <h1 className="text-lg font-bold">All Messages</h1>
+      </div>
+      <div className="flex-none flex flex-col items-start overflow-y-auto p-4">
+        {messages?.map((message, index) => (
+          <div
+            key={index}
+            className={`relative rounded p-4 mb-4 max-w-content ${
+              message.senderId === user.id ? "bg-teal-200 bg-opacity-75 self-end" : "bg-gray-200 bg-opacity-75 self-start"
+            }`}
+          >
+            <div>{message.messageContent}</div>
+            <div className="text-xs text-gray-500 absolute bottom-1 right-1">
+              {formatTimestamp(message.timeStamp)}
             </div>
```

Comment:
I am very like how out logout for the conversations looks like, thank you @mahadaali3 
![Screenshot from 2023-09-29 08-44-36](https://github.com/gallery-apps/whats-app2/assets/51173079/9dbf0d1f-d3fc-41a9-955a-6600d2eb7ab6)
@HamzaAli303